docs/introduction.md

```diff
@@ -4,145 +4,62 @@
-  capabilities which may be only suitable for specific user cases.
-  This makes extending such platform to leverage broader Kubernetes ecosystems
-  almost impossible.
-
-- It could be painful to develop new capability based on existing interfaces.
-  The operator design could easily becomes a marathon negotiation in order
-  to meet the abstraction requirement.
-
-In the end, application developers may complain that Kubernetes is hard to use
-and the platform builders may want to help but they cannot do it easily.
+The modern trends of cloud-native technology has been moved towards pursuing consistency of application delivery across clouds and infrastructure with Kubernetes as the common abstraction layer. However, excellent in abstracting away low level infrastructure details, Kubernetes also introduced another layer of complexity to application developers: pods, port exposing, privilege escalation, resource claims, CRD, and so on. We’ve seen such learning curve and the lack of developer facing abstraction have impacted user experience, slowed down the productivity, led to unexpected errors or misconfigurations in production. People start questioning the value of this revolution, "why am I bothered with these many YAML files?".
+
+On the other hand, abstracting Kubernetes is a highly opinionated process, but the resultant abstraction would only make sense had the opinions come from the application developers. This poses platform engineers with a new challenge: 
+- On one hand, they want to build a platform with developer facing abstractions,
+- On the other hand, the way to build these abstractions should be highly extensible so the platform could adapt to developers' opinions efficiently.
+
+Today, most platforms with higher level abstractions tend to be opinionated on their abstractions, implementations and add-on mechanisms. This makes extending such platforms per developers' needs or to wider scenarios almost impossible, not to mention with full leverage of Kubernetes ecosystems. 
+
+In the end, developers complain such platform has too many constraints and slow in response. The platform engineers may want to help but the cost is huge: any simple API change in this system could easily becomes a marathon negotiation around the abstraction design.
 
 ## What is KubeVela?
 
-KubeVela is a framework to help platform builders to easily expose and extend
-Kubernetes capabilities for application management. It is built on top of
-Kubernetes and relieves the pains of both the platform builders and the application
-developers by doing the following:
+For developers, KubeVela is an easy-to-use tool that enables developers to describe and ship their applications to Kubernetes with zero effort. Instead of managing a handful Kubernetes YAML files, a simple docker-compose style [Appfile](./docs/developers/devex/appfile.md) is what all they need, following an application-centric workflow that can be easily integrated with any CI/CD pipeline.
```

Comment:
```suggestion
For developers, KubeVela itself is an easy-to-use tool that enables them to describe and ship their applications to Kubernetes with minimal efforts. Instead of managing a handful of Kubernetes YAML files, a simple docker-compose style [Appfile](./docs/developers/devex/appfile.md) is all they need. KubeVela also comes with an application-centric workflow that can be easily integrated with any CI/CD pipeline.
```